igdiscover/config.py

```diff
@@ -123,6 +123,22 @@ def from_default_path(cls):
 		with open(cls.DEFAULT_PATH) as f:
 			return Config(file=f)
 
+	@staticmethod
+	def update_option(config_path, options_dict={}):
```

Comment:
Why did you provide a default for `options_dict`? Calling the function without `options_dict` at all wouldn’t make sense, right?

Reply:
Right, it's I forgot about this. Fixed.